tests/expose_test.go

```diff
@@ -330,38 +330,79 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			const servicePort = "28017"
 			const serviceName = "cluster-ip-udp-vmi"
 
-			It("[test_id:1535][label:masquerade_binding_connectivity]Should expose a ClusterIP service on a VMI and connect to it", func() {
+			var vmiExposeArgs []string
+
+			var job *batchv1.Job
+
+			BeforeEach(func() {
+				vmiExposeArgs = []string{
+					expose.COMMAND_EXPOSE,
+					"virtualmachineinstance", "--namespace", udpVM.GetNamespace(), udpVM.GetName(),
+					"--port", servicePort, "--name", serviceName, "--target-port", strconv.Itoa(testPort),
+					"--protocol", "UDP",
+				}
+			})
+
+			AfterEach(func() {
+				Expect(job).NotTo(BeNil(), "a successful test must have stored a way to delete the batchv1.Job entity")
+				Expect(cleanupJob(job.GetName(), job.GetNamespace())).To(BeEmpty(), "should be able to delete the multiple k8sv1.`Job` entities")
+			})
+
+			AfterEach(func() {
+				Expect(cleanupService(serviceName, udpVM.GetNamespace())).To(Succeed(), "should be able to delete the k8sv1.Service entity")
+			})
+
+			table.DescribeTable("[test_id:1535][label:masquerade_binding_connectivity]Should expose a ClusterIP service on a VMI and connect to it", func(ipFamily k8sv1.IPFamily) {
```

Comment:
Please move the test id to the ipv4 entry and skip the ipv6 test in case the cluster is not dual stack. Relevant to all the tests on the commit.

Reply:
same as https://github.com/kubevirt/kubevirt/pull/4272/commits/b3f10f3697674f74bd8763385ef93ec16d4027cf#r511586890